src/common/src/catalog/mod.rs

```diff
@@ -125,6 +125,57 @@ impl From<TableId> for u32 {
     }
 }
 
+#[derive(Clone, Copy, Debug, Display, Default, Hash, PartialOrd, PartialEq, Eq, Ord)]
+pub struct TableIdx {
+    pub table_id: u32,
```

Comment:
Why do we have an `id` inside the `Index` struct? 🤔

Reply:
thank you for the catch. That was a copy-paste error.